app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,48 @@
+class NotesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_note, only: [:show, :edit, :destroy, :update]
+
+  def index
+    @notes = current_user.notes
+  end
+
+  def new
+    @note = current_user.notes.new
+  end
+
+  def create
+    @note = current_user.notes.create(note_params)
+    if @note.save
+      redirect_to user_note_path(current_user, @note), notice: "Note has successfully created."
+    else
+      redirect_to new_notes_path, alert: @note.errors.full_messages
```

Comment:
we should use render in case of failure. 
Replace redirect with render and see the diff

Reply:
Ammar bhae says not use render.